duckduckgo/iOS
# Autofill prompts updated for iOS 16 dynamic sizing

Task/Issue URL: https://app.asana.com/0/72649045549333/1202518616827752/f
Tech Design URL:
CC:

**Description**:
Updates the autofill login prompts (save, update, fill, password generation) to use dynamic sizes based on their content + styling updated to use DesignResourceKit colors & fonts



**Steps to test this PR**:

Trigger the various autofill login prompts on iPhone and iPad validating the layout matches [figma](https://www.figma.com/file/p4apqKWG87wftLau9zgQEq/LOGINS%2B?type=design&node-id=5154%3A359910&t=uHoU2Ev5BBPapAwO-1): 
1. Save login 
2. Save password (https://news.ycombinator.com/login is a good site for testing this)
3. Update username
4. Update password
5. Fill login (maybe try a few permutations of exact and partial matches. You should only ever see the 'More Options' button on iOS 15)
6. Generate password

Repeat the above on iOS 15 (prompts should all look the same as they did on iOS 16 before this update) & iOS 14 (all prompts are full screen - no change)



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
